scripts/components/QuizDetailsComponent.js

```diff
@@ -0,0 +1,42 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var Backbone = require('backbone');
+var QuizModel = require('Models/QuizModel');
+var startTime = new Date();
+
+module.exports = React.createClass({
+	getInitialState: function() {
+	    return {
+	         allQuestions: [],
+	         currentType: null
+	    };
+	   
+	},
+
+	componentWillMount: function() {
+		var queryQuizModel = new Parse.Query(QuizModel);
+
+		queryQuizModel.find().then( 
+			(question) => { 
+				this.setState({allQuestions: question});
+			}
+		);
+	},
+
+	render: function() {
+		if(this.state.allQuestions.length == 0){
+			return (<div></div>);
+		}
+
+		var Question = this.state.Question;
```

Comment:
Where are you defining this.state.Question and this.state.StudentAnswer? I don't see it in your initial state. Also be sure to add a comment to the top of the component explaining what its purpose is and any properties that you're expecting to be passed in.